internal/controller/utils.go

```diff
@@ -187,19 +197,19 @@ func getVolumeMountsForContainer(ctx context.Context, msvc *msv1alpha1.ModelServ
 	uriType := UriType(msvc.Spec.ModelArtifacts.URI)
 
 	switch uriType {
-	case PVC:
-		// don't need the parts
-		if _, err := parsePVCURI(&msvc.Spec.ModelArtifacts); err == nil {
-			desiredVolumeMount = &corev1.VolumeMount{
-				Name:      modelStorageVolumeName,
-				MountPath: modelStorageRoot,
-				ReadOnly:  true,
-			}
-		} else {
-			log.FromContext(ctx).V(1).Error(err, "uri type: "+msvc.Spec.ModelArtifacts.URI)
+
+	// The volume mount for HF and PVC is the same
+	// volumeMounts:
+	// - mountPath: /model-cache
+	//   name: model-storage
+	case PVC, HF:
+		desiredVolumeMount = &corev1.VolumeMount{
+			Name:      modelStorageVolumeName,
+			MountPath: modelStorageRoot,
+			ReadOnly:  true,
```

Comment:
Do we want `ReadOnly: true` for HF? In this case, we want vllm to be able to download a model to the volume.

Reply:
The same could be said about PVC volumes as well. The general use case is there is a model already on the volume. But we could allow the user to let it get written the first time.

Reply:
This sounds like a good default.